bestbook/static/build/js/index.js

```diff
@@ -1,3 +1,5 @@
+var api_url = '//' + window.location.hostname + '/api';
```

Comment:
```suggestion
var api_url = '//' + window.location.host + '/api';
```
`window.location.hostname` does not include the port number, and will fail for folks developing locally.